matching-service/model/match-model.js

```diff
@@ -0,0 +1,33 @@
+import mongoose from 'mongoose';
+
+var Schema = mongoose.Schema;
+let MatchModelSchema = new Schema({
+    matchId: {
+        type: String,
+        required: true,
+        unique: true,
+    },
+    username1: {
+        type: String,
+        required: true,
+    },
+    username2: {
+        type: String,
+        required: true,
+    },
+    difficulty: {
+        type: String,
+        required: true,
+    },
+    questionId: {
+        type: String,
+        required: true,
+    },
+    isActive: {
+        type: Boolean,
+        required: true,
+        default: true,
+    }
```

Comment:
i am thinking perhaps we can determine the active status of a match by simply checking whether it is in redis, then only add a match to DB after a match has ended. this way we can avoid an `isActive` flag, i think we can try to implement this in a later push.